pages/users-governance/reading_new_market_proposal.md

```diff
@@ -0,0 +1,105 @@
+        }
+      },
+      {
+        "@type": "/dydxprotocol.perpetuals.MsgCreatePerpetual",
+        "authority": "dydx10d07y265gmmuvt4z0w9aw880jnsr700jnmapky",
+        "params": {
+            "atomic_resolution": -10,
+            "default_funding_ppm": 0,
+            "id": 1001,
+            "liquidity_tier": 0,
+            "market_id": 1001,
+            "ticker": "BTC-USD"
+        }
+      },
+      {
+        "@type": "/dydxprotocol.clob.MsgCreateClobPair",
+        "authority": "dydx10d07y265gmmuvt4z0w9aw880jnsr700jnmapky",
+        "clob_pair": {
+            "id": 1001,
+            "perpetual_clob_metadata": {
+              "perpetual_id": 1001
+            },
+            "quantum_conversion_exponent": -9,
+            "status": "STATUS_INITIALIZING",
+            "step_base_quantums": 1000000,
+            "subticks_per_tick": 100000
+        }
+      },
+      {
+        "@type": "/dydxprotocol.delaymsg.MsgDelayMessage",
+        "authority": "dydx10d07y265gmmuvt4z0w9aw880jnsr700jnmapky",
+        "msg": {
+            "@type": "/dydxprotocol.clob.MsgUpdateClobPair",
+            "authority": "dydx1mkkvp26dngu6n8rmalaxyp3gwkjuzztq5zx6tr",
+            "clob_pair": {
+                "id": 1001,
+                "perpetual_clob_metadata": {
+                  "perpetual_id": 1001
+                },
+                "quantum_conversion_exponent": -9,
+                "status": "STATUS_ACTIVE",
+                "step_base_quantums": 1000000,
+                "subticks_per_tick": 100000
+            }
+        },
+        "delay_blocks" : 3600
+      }
+    ]
+  }
+```
+
+### Proposal Values
+The following values are visible in a new market proposal.
+
+| Name | Field | Description | Proposer Input |
+| ---- | ---- | ------------------- | -------- |
+| Reference Price | `reference_price` | Starting price of the proposed asset on the exchange. A user proposing a new market should set this based on the price of the asset at the time they create the proposal. | yes |
+| Liquidity Tier | `liquidity_tier` | Liquidity tier of the proposed asset. A user proposing a new market should set this based on [dYdX liquidity tier guidelines](../users-governance/functionalities#liquidity-tiers). | yes |
+| Atomic Resolution | `atomic_resolution` | Precision of the size of the coin.  | no |
+| Minimum Exchanges | `min_exchanges` | Number of exchanges required to list this asset. | no |
+| Minimum Price Change PPM | `min_price_change_ppm` | The minimum price change that causes the oracle price to update. | no |
+| Exponent | `exponent` | Number of decimal places to use to show prices.  | no |
+| Step Base Quantums | `step_base_quantums` | Minimum amount by which you can increase or decrease an order. Same as `step_size`. | no |
+| Subticks Per Tick | `subticks_per_tick` | TODO Determines the subticks per tick. | no |
+| Quantum Conversion Exponent | `quantum_conversion_exponent` | TODO Determines the subtick size. | no |
```

Comment:
Please remind me what quantum_conversion_exponent represents/ point me to where I can find this info 🙏